src/mlpack/core/dists/laplace_distribution.hpp

```diff
@@ -14,8 +14,9 @@
 #ifndef MLPACK_CORE_DISTRIBUTIONS_LAPLACE_DISTRIBUTION_HPP
 #define MLPACK_CORE_DISTRIBUTIONS_LAPLACE_DISTRIBUTION_HPP
 
+#include <mlpack/prereqs.hpp>
 namespace mlpack {
```

Comment:
```suggestion
#include <mlpack/prereqs.hpp>

namespace mlpack {
```
Small spacing fix. :+1: